spec/build.test.js

```diff
@@ -0,0 +1,16 @@
+import * as fs from "fs";
+import { exec, execSync } from "child_process";
+
+
+describe("npm run build", () => {
+  afterEach(() => {
+    exec("git restore ./build ./tokens");
```

Comment:
I'm a little concerned there might be unintentional consequences if we run git restore for all tests. 

Can you describe what would happen if I were in the middle of development and I ran this test locally?

Reply:
@panzacoder if you ran this locally, any uncommitted changes that were made to the tokens.json file, or any uncommitted changes made to the output files in the build directory would be lost and these files would revert to their status at their last commit.

I agree that this is worry-some  and is probably not best practice. However, it is my understanding that we really shouldn't be working on these files in development. The tokens should only be updated by the figma plugin, and the files in the build directory are generated by our build script. 

I was more worried with the unintended consequences of triggering the build and having all the generated outputs just hanging around. It's kind of an odd thing to test 🤷

Reply:
I'm thinking if we don't want these files to be tracked at all in _this_ repo, we should add these folders to the .gitignore before they are created at all, then we can just `rm -rf` those directories after to make sure there's no clutter.

Reply:
@panzacoder Ya that makes sense. I don't think we need the build files to be tracked in this repo. The only file we do need to keep is a source token file - it's needed for the token transformer & for our test to run properly. Reconfigured the output of the token transformer to a separate directory which is now ignored & cleaned up after the test runs.